schema-engine/connectors/mongodb-schema-connector/src/sampler/statistics.rs

```diff
@@ -153,6 +153,11 @@ impl<'a> Statistics<'a> {
                         let comment = "This collection uses a JSON Schema defined in the database, which requires additional setup for migrations. Visit https://pris.ly/d/mongodb-json-schema for more info.";
                         model.documentation(comment)
                     }
+
+                    if walker.is_capped() {
+                        let comment = "This collection is marked as capped, which requires [a good message here]. Visit https://pris.ly/d/mongodb-capped-collections for more info";
```

Comment:
If anyone has thoughts here on how to word this.
My initial thought felt maybe a little off?
```suggestion
                        let comment = "This collection is marked as capped, which requires further support in Prisma Client. Visit https://pris.ly/d/mongodb-capped-collections for more info";
```

Reply:
Maybe like this?
```suggestion
                        let comment = "This model is a capped collection, which requires manual setup when creating the collection in MongoDB: https://pris.ly/d/mongodb-capped-collections";
```

Reply:
Hm, it feels like it shouldn't necessarily just be specific to creation of the collection in Mongo though. Like it needs both the manual mongo mql and the client work-flow with `runCommandRaw`

Reply:
That is true. Probably something super generic "which requires additional attention to be usable with Prisma" maybe.

Reply:
Updated this as follows
```suggestion
let comment = "This model is a capped collection, which requires further support to be used with Prisma: https://pris.ly/d/mongodb-capped-collections";```